src/main/java/com/teragrep/aer_02/json/JsonRecords.java

```diff
@@ -78,9 +78,14 @@ public String[] records() {
             return rv;
         }
 
-        final JsonValue recordsStructure = mainStructure.getValue("/records");
```

Comment:
Was there a reason why this has a slash before the "records"?

Reply:
getValue() uses JSONPointer, and it uses `/` as a separator. Like `/list/0` would fetch first item in list.
get() just directly accesses the underlying map, returning null if no key is present.

Reply:
Thanks for letting me know about this change!